server/routes/transport.js

```diff
@@ -28,13 +29,13 @@ const signIn = async (req, res) => {
     } else if (token === '404') {
       console.log('No record matches');
       res.status(401).send({ existing: false });
+    } else if (token === '500') {
+      console.error('User name input empty');
+      res.sendStatus(500).send({ input: false });
     } else if (token) {
       res.setHeader('Set-Cookie', cookie.serialize('AccessToken', token, { httpOnly: true }));
       res.status(200).send({ existing: true });
     }
-  } else {
```

Comment:
I think having a catch-all `else` is good practice.  It's where you can send a 500 and say `unknown error occurred` and log that error